### ISSUE TITLE
Superfluous, but useful information

### ISSUE THREAD

**Issue by adammontville** [adammontville/cis-controls-71-measures#182]
If we include superfluous measures (i.e. measures that are not relied upon by a given metric) we should move them out into an "optional measures" (or similar) section for the given sub-control, so that it is clear that the measures are not a requirement.

**Comment by apiperCIS** [comment]
A few comments on this:
1) With regard to measures vs. metrics, my focus has been to ensure that all the needed measures are specified and available.  Then, whoever implements the specification (tool vendors for example) can build desired metrics from those measures.  While we provide example metrics in the metrics section, it is easy to imagine that additional or other metrics would be preferred.  To me, it seems like getting the needed measures right is the higher priority - and then, these measures can be used as building blocks (but we don't need to imagine or specify every possible thing that can be built with those blocks).  By this logic, the fact that there isn't currently a metric making use of a particular measure, doesn't necessarily mean that the measure is superfluous.
2) Related to this, a general goal I have tried to follow when selecting metrics is that the score from the metric could be considered a general score for the Sub-Control (for instance, you have 75/100 done right, so you have a 75% for that Sub-Control), so this is generally the positive (compliant) score.  There are cases where the negative frame of reference might be preferred instead (the non-compliant score); for example, 25/100 are not meeting that Sub-Control.  But, I don't know that we need to list both frames of reference for each and every Sub-Control in CAS.
3) With regard to the non-compliant lists as a measure (we had discussed this some before) - I think it is worthwhile to require the non-compliant list (including why those on the list are non-compliant), and not just the number of items that are non-compliant because this non-compliant list would be useful to organizations try to fix the non-compliant items.
4) A note on notation - the cardinality of the set notation has been suggested here; that is, using the absolute value signs to indicate the size of a list, such as |M1| .  The benefit would be a shorter list of measures - you wouldn't have to have both the list and the count as separate measures.  This works fine for simple sets such as M1 = {1, 4, 7, 3, 2} so |M1| = 5.  But, for more complex sets like we frequently encounter in our measurement proposals, I think keeping the counts specified as separate measures provides greater clarity.  For example, if Input 1 is the hardware inventory, that hardware inventory has many fields beyond just the asset's identifier.  So, it seems clearer to me to have a measure that says "Count of hardware assets in the hardware inventory" rather than just saying "|Input 1|" so that we're specifying what is being counted.
